reference/constraints/GreaterThan.rst

```diff
@@ -23,8 +23,10 @@ than another value, see :doc:`/reference/constraints/LessThan`.
 Basic Usage
 -----------
 
-If you want to ensure that the ``age`` of a ``Person`` class is greater
-than ``18``, you could do the following:
+The following constraints ensure that:
+- the number of ``siblings`` of a ``Person`` is greater than ``5``
```

Comment:
Please leave a blank line before the list items. Otherwise, it will be displayed like this:

![rst_syntax_error](https://cloud.githubusercontent.com/assets/73419/16340973/174eb588-3a2b-11e6-9b25-972d1a59e4d7.png)